unity/Assets/Scripts/Unity/UnityPossiblePoints.cs

```diff
@@ -0,0 +1,41 @@
+﻿using UnityEngine;
+using TMPro;
+using Unity;
+
+namespace Hexxle.Unity
+{
+    public class UnityPossiblePoints : MonoBehaviour
+    {
+        public TMP_Text possiblePointsText;
+        public int possibleScore { get; set; } = 0;
+        public Color32 positiveColor = new Color32(102, 193, 40, 255);
+        public Color32 negativeColor = new Color32(255, 48, 40, 255);
+        private Color32 originalColor;
+
+        private void Awake()
+        {
+            originalColor = possiblePointsText.color;
+        }
+
+        void FixedUpdate()
```

Comment:
Is there a reason for using `FixedUpdate` over regular `Update`? I did a quick google search, and to me it doesn't seem like it would make a difference anywy, so I'm just curious.

Reply:
`FixedUpdate` is, according to the [documentation](https://docs.unity3d.com/ScriptReference/MonoBehaviour.FixedUpdate.html), used for physics-based calculations (or well, supposed to), as the calls are constant and not variable like `Update` that is based on fps. 

It also has a neat feature: We can wait for its update explicitely in tests. There is a problem during tests where the "score" itself would never update, which was solved using [`WaitForFixedUpdate()`](https://docs.unity3d.com/ScriptReference/WaitForFixedUpdate.html) and moving the render logic to `FixedUpdate`.